src/middleware/queries/settlements.ts

```diff
@@ -53,6 +54,45 @@ const instructionAttributes = `
 
 type InstructionArgs = 'id' | 'venueId' | 'status';
 
+/**
+ * Filters for instructions
+ *
+ */
+export interface InstructionPartiesFilters {
+  /**
+   * The DID of the identity to filter by
+   */
+  identity?: string;
+  /**
+   * The asset ID to filter by
+   */
+  assetId?: string;
```

Comment:
```suggestion
  asset?: (string | Asset);
```
Similar to identity, we need to add support for both.